### ISSUE TITLE
error copy file to dest: ***, error message: Process exited with status 1

### ISSUE THREAD

**Issue by flightcode** [appleboy/scp-action#8]
I'm getting an issue where I can't deploy the file because of the issue above. I have created an ssh user, and the folder I am trying to push to is a subdirectory of the user's home. I have tried giving the user root perms, but not working.

**Comment by Deph0** [comment]
Why was this closed? Was it fixed for you? if so how?

I am getting the same issue trying to deploy a folder

> tar all files into /tmp/013702742/5nMYEpit3G.tar
scp file to server.
2020/04/01 16:21:18 error copy file to dest: ***, error message: Process exited with status 1
drone-scp error:  error copy file to dest: ***, error message: Process exited with status 1

**Comment by ivanberry** [comment]
> Why was this closed? Was it fixed for you? if so how?
> 
> I am getting the same issue trying to deploy a folder
> 
> > tar all files into /tmp/013702742/5nMYEpit3G.tar
> > scp file to server.
> > 2020/04/01 16:21:18 error copy file to dest: ***, error message: Process exited with status 1
> > drone-scp error:  error copy file to dest: ***, error message: Process exited with status 1

Did you fix this problem?

**Comment by Deph0** [comment]
> > Why was this closed? Was it fixed for you? if so how?
> > I am getting the same issue trying to deploy a folder
> > > tar all files into /tmp/013702742/5nMYEpit3G.tar
> > > scp file to server.
> > > 2020/04/01 16:21:18 error copy file to dest: ***, error message: Process exited with status 1
> > > drone-scp error:  error copy file to dest: ***, error message: Process exited with status 1
> 
> Did you fix this problem?

No, i gave up.

**Comment by saqibameen** [comment]
@ivanberry what did you do about it?

**Comment by zreren** [comment]
I encountered this issue where I was unable to copy files to a remote server via SCP. Initially, after confirming that permissions were not the problem, I sought to diagnose the issue further. I decided to use the df command to inspect the disk space utilization, and it turned out that there was very little free disk space left.

Subsequently, I used the command find / -type f -size +100M -exec du -h {} \; to identify which files were consuming a significant amount of space. I discovered several unnecessary files, which I promptly deleted. After performing this cleanup, I attempted the SCP copying operation again, and this time it succeeded.
